cluster.go

```diff
@@ -391,3 +391,17 @@ func (c *Cluster) Close() error {
 
 	return err
 }
+
+// Stats returns the current statistics for all pools.
```

Comment:
Could you add a line indicating that the statistics are keyed by the cluster node's address?